src/mlpack/tests/nbc_test.cpp

```diff
@@ -26,8 +26,10 @@ TEST_CASE("NaiveBayesClassifierTest", "[NBCTest]")
   size_t classes = 2;
 
   arma::mat trainData, trainRes, calcMat;
-  data::Load(trainFilename, trainData, true);
-  data::Load(trainResultFilename, trainRes, true);
+  if (!data::Load(trainFilename, trainData, true)) 
+    FAIL("Cannot load dataset trainSet.csv"); 
+  if (!data::Load(trainResultFilename, trainRes, true)) 
```

Comment:
```suggestion
  if (!data::Load(trainResultFilename, trainRes)) 
```
Third argument not required.